e2e-tests/cypress/support/appellant-submission-decision-letter/confirmDecisionLetterFileIsNotUploaded.js

```diff
@@ -0,0 +1,5 @@
+module.exports = () => {
+  cy.goToAppellantSubmissionDecisionLetter();
+  cy.get('#upload-decision-file-name').should('not.exist');
```

Comment:
for these css selectors, have you come across the data-cy="xxx" way of marking things out?
it's the way cypress suggests you do things; specifically putting the tag into the element you want to work against, that way people can re-style whatever and so long as they leave the tag in: the tests don't care.
This isn't always -easy- with govuk components but i've put examples in for some of my tests where it was easy + just put the tag into the page while i was writing the test.

Reply:
This is the issue I raised at standup. These identifier are already in use so the tests are playing catchup. I would prefer a naming convention that makes clear something is important for testing.